evm/src/keccak_sponge/keccak_sponge_stark.rs

```diff
@@ -358,8 +358,8 @@ impl<F: RichField + Extendable<D>, const D: usize> KeccakSpongeStark<F, D> {
         row.updated_state_u32s = sponge_state.map(F::from_canonical_u32);
         let is_final_block = row.is_final_input_len.iter().copied().sum::<F>() == F::ONE;
         if is_final_block {
+            let mut cur_bytes = vec![F::ZERO; 4];
```

Comment:
Seems like `cur_bytes` isn't really needed, could just inline it

Reply:
Good point, just addressed it.